lib/components/banner/banner.less

```diff
@@ -37,8 +38,7 @@
     inset: 0 0 auto 0;
     padding: var(--su12);
     position: fixed;
-    -webkit-transform: translate3d(0, calc(var(--theme-topbar-height, calc(var(--su-static48) + var(--su-static8))) * var(--_no-x-offset)), 0);
```

Comment:
@dancormier I dropped the vendor prefix since I believe we don't support old version of browsers. Let me know if that makes sense.

Reply:
Makes sense!